### PR TITLE
add s3 bucket resource to interventions dev environment

### PR DESCRIPTION
**Pull request by tomsmyers**
i copy pasted this code from elsewhere in this repo. is this everything that is required to get this up and running?

thanks!

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/s3.tf [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  providers = {
+    # Can be either "aws.london" or "aws.ireland"
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "interventions_s3_bucket" {
+  metadata {
+    name      = "interventions-s3-bucket"
```
Comment (sldblog) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/s3.tf:24]: As this secret will be retrieved via `kubectl get secret/interventions-s3-bucket --namespace=hmpps-interventions-dev`, `interventions` seems redundant. What do you think about?
```suggestion
    name      = "storage-s3-bucket"
```

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/s3.tf [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/*
+ Based on https://github.com/ministryofjustice/cloud-platform-terraform-s3-bucket/tree/main/example
+ */
+module "interventions_s3_bucket" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=4.6"
+
+  team_name              = var.team_name
+  business-unit          = "Digital and Technology"
```
Comment (sldblog) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/s3.tf:8]:
```suggestion
  business-unit          = var.business_unit
```

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/s3.tf (hunk 1 above)
Comment (tomsmyers) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/s3.tf:24]: works for me 👍

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/s3.tf (hunk 2 above)
Comment (tomsmyers) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/s3.tf:8]: 👍

**Review by sldblog — APPROVED** [review]
LGTM, thanks :)

**Review by jasonBirchall — APPROVED** [review]

**Pull request merged**